add_pypi_package.sh

```diff
@@ -77,10 +77,12 @@ add_to_tito_props() {
 
 add_pypi_to_comps() {
   local comps_scl="nonscl"
-  local comps_package="${PACKAGE_NAME}"
+  local comps_packages=$(rpmspec --query --builtrpms --queryformat '%{NAME}\n' $PACKAGE_DIR/*.spec)
```

Comment:
Can there be macros so it either outputs Python 2 or Python 3 depending on the host platform? For example, when I run it on Fedora, can it still output Python 2 for EL7 if needed?

Reply:
"hah" (no)